doc/spec/2025-11-26_ユーザー情報アクセス制御.md

```
@@ -0,0 +1,305 @@
+
+| 環境変数 | 役割 | 用途 |
+|----------|------|------|
+| `E2E_CLERK_USER` / `E2E_CLERK_PASSWORD` | owner | メインユーザー（既存） |
+| `E2E_CLERK_USER_MANAGER` / `E2E_CLERK_PASSWORD_MANAGER` | manager | マネージャーテスト用 |
+| `E2E_CLERK_USER_GENERAL` / `E2E_CLERK_PASSWORD_GENERAL` | general | 一般スタッフテスト用 |
+
+※ `E2E_CLERK_USER_B` は別用途のため除外
+
+### fixture構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)